From 41fc972095df090f559793317822b46da53fdaf5 Mon Sep 17 00:00:00 2001
From: Jeeja KP <jeeja.kp@intel.com>
Date: Tue, 20 Oct 2015 00:40:38 +0530
Subject: [PATCH 1741/2508] ASoC: Intel: Skylake: Fixed compilation warning in
 debugfs

commit 80bf6a64b731df725e8f5415c53354be8007c447 from
https://github.com/01org/linux-apollolake-i

warning: passing argument 1 of skl_print_fmt incompatible pointer
type [enabled by default]
note: expected struct skl_module_fmt *a argument is of type a sruct
skl_module_fmt
warning: passing argument 1 of skl_print_fmt *a incompatible pointer
type [enabled by default]

Signed-off-by: Jeeja KP <jeeja.kp@intel.com>
---
 sound/soc/intel/skylake/skl-debug.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/sound/soc/intel/skylake/skl-debug.c b/sound/soc/intel/skylake/skl-debug.c
index 004ce82..acb0fcb 100644
--- a/sound/soc/intel/skylake/skl-debug.c
+++ b/sound/soc/intel/skylake/skl-debug.c
@@ -235,8 +235,8 @@ static ssize_t module_read(struct file *file, char __user *user_buf,
 			mconfig->max_out_queue,
 			mconfig->is_loadable ? "loadable" : "inbuilt");
 
-	ret += skl_print_fmt(&mconfig->in_fmt, buf, ret, true);
-	ret += skl_print_fmt(&mconfig->out_fmt, buf, ret, false);
+	ret += skl_print_fmt(mconfig->in_fmt, buf, ret, true);
+	ret += skl_print_fmt(mconfig->out_fmt, buf, ret, false);
 
 	ret += snprintf(buf + ret, MOD_BUF - ret,
 			"Fixup:\n\tParams %x\n\tConverter %x\n",
-- 
1.7.5.4

